torchvision/prototype/transforms/_container.py

```diff
@@ -39,6 +42,9 @@ def __init__(self, transforms: Union[Sequence[Callable], nn.ModuleList], p: floa
             raise ValueError("`p` should be a floating point value in the interval [0.0, 1.0].")
         self.p = p
 
+    def _extract_params_for_v1_transform(self) -> Dict[str, Any]:
+        return {"transforms": self.transforms, "p": self.p}
```

Comment:
I don't understand why, but the `transforms` key was missing when using `_extract_params_for_v1_transform()` from the base class.

Reply:
`nn.Module` overwrites `__setattr__`. One of the things it is doing is to not assign other `nn.Module`'s into `self.__dict__`, but rather into `self._modules`. Since we only go through `self.__dict__`, we don't pick up `self.transforms` since `isinstance(nn.ModuleList(...), nn.Module)`.

Reply:
wow, good catch! thanks for the explanation